dashboards/TDinsight.json

```diff
@@ -2824,139 +2824,6 @@
       },
       "id": 108,
       "panels": [
```

Comment:
bump the version  in the end of the file.

    "version": 27

to next version:

    "version": 28

Reply:
what is the meaning of schemaVersion? need to modify this to 28?